Composer/packages/client/src/components/CellFocusZone.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import React, { useCallback } from 'react';
+import { FocusZone, FocusZoneTabbableElements, IFocusZoneProps } from 'office-ui-fabric-react/lib/FocusZone';
+import { getFocusStyle, getTheme, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+export const formCell = css`
+  outline: none;
+  white-space: pre-wrap;
+  font-size: 14px;
+  line-height: 28px;
+`;
+
+export const formCellFocus = mergeStyles(
+  getFocusStyle(getTheme(), {
+    inset: -3,
+  })
+);
+
+/**
+ * CellFocusZone component props.
+ * Props are intently omitted as they're required for component to function correctly.
+ * Please manually test the component using keyboard in places it is used in case any changes made.
+ */
+type CellFocusZoneProps = Omit<
+  React.HTMLAttributes<unknown> & IFocusZoneProps,
+  | 'allowFocusRoot'
+  | 'data-is-focusable'
+  | 'isCircularNavigation'
+  | 'className'
+  | 'handleTabKey'
+  | 'shouldFocusInnerElementWhenReceivedFocus'
+  | 'tabIndex'
+  | 'onBlur'
+  | 'onKeyDown'
+>;
```

Comment:
Can you briefly explain the reason for this?

Reply:
By limiting these props from being changed we ensure focus behavior is consistent for tables using the component. We may want to make the component even more non-configurable i.e. by not specifying `IFocusZoneProps` at all (in this case focus zone props can't be changed from the outside of the component).

Decided to go with a more relaxed version for now

Reply:
Ok, that sounds good to me.